activerecord/lib/active_record/associations/builder/belongs_to.rb

```diff
@@ -121,6 +121,13 @@ def self.define_validations(model, reflection)
         required = !reflection.options[:optional]
       end
 
+      case reflection.options[:polymorphic]
+      when Array, String, Symbol
```

Comment:
Should we accept String and Symbol? If there is only one possible value people should just pass an array with one element

Reply:
Are you thinking we simply support arrays of strings? I was trying to be flexible, but only supporting an array of strings is certainly easier to document and explain.

Reply:
Or are you saying I just support an array, and then map the values to string still (which would allow an array of strings/symbols)?

Reply:
I think Rafael is suggesting the latter: just support `Array[String|Symbol]` and not support passing a single string or symbol.

Reply:
I've made the changes, but I'm not sure what to do about the changelog. I see @rafaelfranca changed the milestone to 8.0.0. So I need to rebase this to main's recently wiped changelog instead of 7.2.0's changelog?

Reply:
@natematykiewicz Yes, please do add it to the new (empty or nearly-empty) CHANGELOG file, this would now land in 8.0.

Reply:
Done @flavorjones!